### DISCUSSION TITLE
ARK: Survival Evolved - Linux Install - Any Tools or Management Software Available?

### DISCUSSION THREAD

**Discussion by Bosnis** [GameServerManagers/LinuxGSM#4654] [General]
Hi, 
Using LGSM I have ARK Survival Evolved installed on my linux box all up and running fine. 
Are there any tools or management softwares available to alter/moderate this game server? 
Editing .ini files in terminals or via ftp is far from convenient. 
Any recommendations?

**Comment by nghialele** [comment]
Brother have you heard of GameServerApp?

**Reply by Bosnis** [reply]
Hello friend, thanks for the recommendation, I did actually recently find this but haven't tried it yet.  Have you used it?

**Reply by nghialele** [reply]
Hi, I am using it for one of my Rust server, still exploring their product but so far it’s great.
AFAIK, they have great support for ARK gameservers, even LGSM recommend them in the installation section.

Also, I’m now interesting in ARK 😆
